recipe/meta.yaml

```diff
@@ -11,13 +11,18 @@ source:
 
 build:
     number: 0
+    script:
+        - python setup.py build_ext --inplace --compiler=mingw32  # [win and py<35]
+        - python setup.py install --skip-build  # [win and py<35]
+        - python setup.py install --single-version-externally-managed --record record.txt  # [not (win and py<35)]
```

Comment:
I believe that is a limitation on `vc9` (used for Python 2.7) and `vc10` (for Python 3.4) that no longer exist on `vc14` (Python 3.5). So I am not sure that is possible.

Let's hope `mingwpy` fixes the builds for `[win and py<35]`, if not we can skip Python 2.7 and 3.4 on Windows and tell people to use 3.5 :grimacing: